local_costmap:
  robot_base_frame: base_link #nazwa podstawy robota
  global_frame: odom 
  update_frequency: 10.0 #częstotliwość odświeżania mapy
  publish_frequency: 2.0 #częstotliwość publikowania parametrów
  static_map: false #brak wykorzystania mapy statycznej, czyli tej z lab3
  rolling_window: true
  width: 5.0 #szerokość lokalnej trasy
  height: 5.0 #wysokość lokalnej trasy
  resolution: 0.05 #przeskalowanie do pikseli

  plugins: 
    - {name: obstacles,        type: "costmap_2d::VoxelLayer"}
    - {name: inflation,        type: "costmap_2d::InflationLayer"}
  obstacles:
    observation_sources: base_scan #przeszkody publikowane na topicu /base_scan
    base_scan: {data_type: LaserScan, sensor_frame: base_laser_link, clearing: true, marking: true, topic: /scan_raw}
  inflation:
    inflation_radius: 0.3 #wartośc promienia od przeszkód
